client.go

```diff
@@ -159,6 +168,76 @@ func (c *Client) CheckAckStatus(ctx context.Context, channel, topic, id string)
 	return MessageS(m).ToMessage(), nil
 }
 
+//go:embed ui
+var views embed.FS
+
+func (c *Client) ServeHttp(ctx context.Context) {
+
+	go func() {
+		timer := timex.TimerPool.Get(10 * time.Second)
+		defer timer.Stop()
+
+		for range timer.C {
+			select {
+			case <-ctx.Done():
+				return
+			default:
+
+			}
+			timer.Reset(10 * time.Second)
+			if err := c.broker.tool.QueueMessage(ctx); err != nil {
+				logger.New().Error(err)
+			}
+		}
+
+	}()
+
+	httpport := ":9090"
```

Comment:
Why the port number is hard-coded?

Reply:
Thank you for your reminder. 
I will move it into env.json.

Reply:
Yes. 
We should maintain a different section for UI settings. One section for beanQ service and another for beanQUI.